dvc/updater.py

```diff
@@ -72,8 +72,14 @@ def fetch(self, detach=True):
     def _get_latest_version(self):
         import json
 
-        r = requests.get(self.URL, timeout=self.TIMEOUT_GET)
-        info = r.json()
+        try:
+            r = requests.get(self.URL, timeout=self.TIMEOUT_GET)
+            info = r.json()
+        except requests.exceptions.RequestException as exc:
+            msg = "Failed to retrieve latest version: {}"
+            Logger.debug(msg.format(exc))
```

Comment:
So, if I run `dvc init -v` it will still show the message, right?

The problem was that it shows an error unrelated to the command I'm running; it was ok when it showed the "Checking for latest version..." but it is hard to be aware that the command is spawning another process.

What do you think, @efiop ? Am I tripping to much :stuck_out_tongue: ?

Reply:
No, it won't, since daemon is spawned in quiet mode by default(see cmd in dvc/daemon.py). It will show the message with `dvc daemon updater -v` though, which is expected. And fatal messages, that will bypass the -q mode, should turn up in the current output(if they can), since they are bugs and at least we will be able to see them.

Reply:
Awesome, @efiop! :+1: